contrib/kind.sh

```diff
@@ -316,7 +321,9 @@ if [ "$OVN_IMAGE" == local ]; then
   sudo cp -f ../../go-controller/_output/go/bin/* .
   echo "ref: $(git rev-parse  --symbolic-full-name HEAD)  commit: $(git rev-parse  HEAD)" > git_info
   docker build -t ovn-daemonset-f:dev -f Dockerfile.fedora .
+  docker build -t ovnkube-admission-controller:dev -f Dockerfile.admission .
```

Comment:
IIUIC this should have its own `if OVN_ADMISSION_IMAGE == local`  and be independent of the OVN_IMAGE

Reply:
Since `ovnkube` and `ovnkube-admission-controller` will practically go hand-in-hand, as I see it, I'd say it's fine to leave it like this for now.

Reply:
my point is if you are developing in one or the other, is it possible that you only one to choose one, or deploy with different versions of each.

Reply:
I am actually more inclined to dropping the `Dockerfile.admission` completely and just stick the binary in the ovnkube image. I really don't think this will be a big use case (deploying different versions)

Reply:
This is also what we will end up doing for OCP

Reply:
Yes, I will do that. It saves us download/upload time during the different stages of our CI runs too.

Reply:
lgtm